cohttp-async/src/server.ml

```diff
@@ -85,7 +85,6 @@ let handle_client handle_request sock rd wr =
                     Http.Request.is_keep_alive req
                     && Http.Response.is_keep_alive res
                   in
-                  let flush = Http.Response.flush res in
```

Comment:
It's not clear to me we want to remove this completely - @rgrinberg what's the intended use for those nowadays?

Reply:
I think they're useless. Flushing should just be done for every request/response.

Reply:
`flush` defaults to true, so removing these lines is ok.